stories/patterns/filters/advanced-filters/docs/index.stories.tsx

```diff
@@ -1,6 +1,5 @@
 import type { Meta, StoryObj } from '@storybook/react';
-
-import FiltersWithConditionsEx from './examples/filters-with-filter-conditions';
+import FiltersWithConditionsEx from './docs-examples/filters-with-filter-conditions';
```

Comment:
incorrect changes

Reply:
Fixed

Reply:
should be just `examples/filters-with-filter-conditions`

Reply:
fixed